command/cp.go

```diff
@@ -557,6 +569,13 @@ func (c Copy) doUpload(ctx context.Context, srcurl *url.URL, dsturl *url.URL) er
 		SetCacheControl(c.cacheControl).
 		SetExpires(c.expires)
 
+	if c.contentType != "" {
+		metadata.SetContentType(c.contentType)
```

Comment:
If `content-type` is given, there is no need to calculate/guess content-type. Could you please remove line [564](https://github.com/peak/s5cmd/blob/330c26cac176bbaad5690ed5c239b04a17b0f345/command/cp.go#L564) and call `guessContentType` only if content-type flag is empty.